Add how to start near node from backup (#720)
near/docs: docs/develop/node/running-a-node.md

Create new instance, with at least:

> * 8 vCPU and 32 GB of RAM (We recommend e2-standard-8).

Reviewer: Why do we need 8 vCPU? 32GB of RAM should not be a requirement as well. Can we be more constraint about our recommendations?

> * 8 vCPU and 32 GB of RAM (We recommend e2-standard-8).

Reviewer 2: 4cpu and 8GB of ram should be enough. Even 2 cpus could be fine

> * 8 vCPU and 32 GB of RAM (We recommend e2-standard-8).

Author: Just took the ones we're using for official rpc nodes, can align them with @bowenwang1996 recommendations

To check logs call: `nearup logs` or `nearup logs --follow`
```

## Starting a node from backup
Using data backups allows you to sync your node quickly by using public tar backup files. There are two types of backups for available for both `testnet` and `mainnet`:
- regular 
- archival 

### Archive links

`mainnet` 
  - [regular](https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/mainnet/rpc/data.tar)
  - [archival](https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/mainnet/archive/data.tar)

`testnet`
  - [regular](https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/testnet/rpc/data.tar)
  - [archival](https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/testnet/archive/data.tar)

Starting node using `neard` backup data 

```bash
./neard init --chain-id <chain-id> --download-genesis
cd ~/.near/data
wget <link-above>
tar -xf data.tar
rm data.tar
./neard run
```

Starting node using `nearup` backup data:

```bash

> nearup run <chain-id> && sleep 30 && nearup stop

Reviewer: `sleep 30` won't be enough to download testnet genesis records, won't this break things?

>Got a question?

> <a href="https://stackoverflow.com/questions/tagged/nearprotocol">

Reviewer 3: The "Got a question" prompt with link to SO should be at the bottom of the document.

  <h8>Ask it on StackOverflow!</h8></a>

## Starting a node from backup

> There are public backups of data available as tar archives.
> This enables you to sync data from the network faster.
> We have regular backups and archival backups.
> We also keep those for two chains, mainnet and testnet.

Reviewer 3: ```suggestion
Using data backups allows you to sync your node quickly by using public tar backup files. There are two types of backups for available for both `testnet` and `mainnet`:
- regular 
- archival 
```

Reviewer 3: Maybe we could add a description for regular and archival so people know what the difference between the two are.

### Archive links

> Mainnet regular data
> 
> https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/mainnet/rpc/data.tar
> 
> Mainnet archival data
> 
> https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/mainnet/archive/data.tar
> 
> Testnet regular data
> 
> https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/testnet/rpc/data.tar
> 
> Testnet archival data
> 
> https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/testnet/archive/data.tar

Reviewer 3: ```suggestion
`mainnet` 
  - [regular](https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/mainnet/rpc/data.tar)
  - [archival](https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/mainnet/archive/data.tar)

`testnet`
  - [regular](https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/testnet/rpc/data.tar)
  - [archival](https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/testnet/archive/data.tar)
```

> 

Reviewer 3: ```suggestion

### Starting the node
```

Author: Solved most of the changes, lost context for this one, could you review this PR again

Reviewer 3: lol... not sure what this is either... disregard :)

> Starting node from backup data neard

Reviewer 3: ```suggestion
Starting node using `neard` backup data 

```

```bash
./neard init --chain-id <chain-id> --download-genesis
cd ~/.near/data
wget <link-above>
tar -xf data.tar
rm data.tar
./neard run
```

> Starting node from backup data nearup

Reviewer 3: ```suggestion
Starting node using `nearup` backup data:

```

```bash
nearup run <chain-id> && sleep 30 && nearup stop
cd ~/.near/<chain-id>/data
rm ./* # clean up old DB files to avoid corruption
wget <link-above>
tar -xf data.tar
rm data.tar
nearup run <chain-id> 
```

> <chain-id> corresponds to testnet or mainnet

Reviewer 3: ```suggestion
( <chain-id> corresponds to `testnet` or `mainnet` )
```

> by default neard stores data in ~/.near/data
> 
> nearup stores data in ~/.near/<chain-id>/data

Reviewer 3: ```suggestion
**Note:** Default location for `neard` data is `~/.near/data`. `nearup` stores data by default in `~/.near/<chain-id>/data.`
```
